diff --git a/node_modules/tiptap-pagination-plus/dist/PaginationPlus.js b/node_modules/tiptap-pagination-plus/dist/PaginationPlus.js
index 52bd368..60f19f5 100644
--- a/node_modules/tiptap-pagination-plus/dist/PaginationPlus.js
+++ b/node_modules/tiptap-pagination-plus/dist/PaginationPlus.js
@@ -59,7 +59,11 @@ export const PaginationPlus = Extension.create({
         return defaultOptions;
     },
     addStorage() {
-        return Object.assign(Object.assign({}, defaultOptions), { headerHeight: new Map(), footerHeight: new Map() });
+          return {
+            ...this.options,
+            headerHeight: new Map(),
+            footerHeight: new Map()
+        }
     },
     onCreate() {
         const targetNode = this.editor.view.dom;
@@ -122,10 +126,6 @@ export const PaginationPlus = Extension.create({
       .rm-with-pagination table tbody > tr{
         display: table-row !important;
       }
-      .rm-with-pagination *:has(>br.ProseMirror-trailingBreak:only-child) {
-        display: table;
-        width: 100%;
-      }
       .rm-with-pagination .rm-br-decoration {
         display: table;
         width: 100%;
@@ -202,36 +202,28 @@ export const PaginationPlus = Extension.create({
                         };
                     },
                     apply: (tr, oldDeco, oldState, newState) => {
-                        const pageCount = calculatePageCount(editor.view, this.options);
-                        const currentPageCount = getExistingPageCount(editor.view);
+                        const currentView = safeGetView(editor);
+                        if (!currentView || !currentView.dom) {
+                            return oldDeco;
+                        }
+
+                        const opts = editor.storage.PaginationPlus
+                        const pageCount = calculatePageCount(currentView, opts);
+                        const currentPageCount = getExistingPageCount(currentView);
+
                         const getNewDecoration = () => {
-                            updateCssVariables(editor.view.dom, this.options);
+                            updateCssVariables(editor.view.dom, opts);
                             let headerHeight = "headerHeight" in this.storage ? this.storage.headerHeight : new Map();
                             let footerHeight = "footerHeight" in this.storage ? this.storage.footerHeight : new Map();
-                            const widgetList = createDecoration(this.options, headerHeight, footerHeight);
-                            this.storage = Object.assign(Object.assign({}, this.options), { headerHeight, footerHeight });
+                            const widgetList = createDecoration(opts, headerHeight, footerHeight);
+                            this.storage = { ...opts, headerHeight, footerHeight }
                             return {
                                 decorations: DecorationSet.create(newState.doc, [...widgetList]),
                                 footerHeight
                             };
                         };
-                        if ((pageCount > 1 ? pageCount : 1) !== currentPageCount ||
-                            this.storage.pageBreakBackground !== this.options.pageBreakBackground ||
-                            this.storage.pageHeight !== this.options.pageHeight ||
-                            this.storage.pageWidth !== this.options.pageWidth ||
-                            this.storage.marginTop !== this.options.marginTop ||
-                            this.storage.marginBottom !== this.options.marginBottom ||
-                            this.storage.marginLeft !== this.options.marginLeft ||
-                            this.storage.marginRight !== this.options.marginRight ||
-                            this.storage.pageGap !== this.options.pageGap ||
-                            this.storage.contentMarginTop !== this.options.contentMarginTop ||
-                            this.storage.contentMarginBottom !== this.options.contentMarginBottom ||
-                            this.storage.headerLeft !== this.options.headerLeft ||
-                            this.storage.headerRight !== this.options.headerRight ||
-                            this.storage.footerLeft !== this.options.footerLeft ||
-                            this.storage.footerRight !== this.options.footerRight ||
-                            !deepEqualIterative(this.options.customHeader, this.storage.customHeader) ||
-                            !deepEqualIterative(this.options.customFooter, this.storage.customFooter)) {
+
+                        if ((pageCount > 1 ? pageCount : 1) !== currentPageCount || !deepEqualIterative(opts, this.storage)) {
                             return getNewDecoration();
                         }
                         return oldDeco;
@@ -246,7 +238,8 @@ export const PaginationPlus = Extension.create({
                 view: (editorView) => {
                     return {
                         update: (view) => {
-                            const pageCount = calculatePageCount(view, this.options);
+                            const opts = editor.storage.PaginationPlus
+                            const pageCount = calculatePageCount(view, opts);
                             const currentPageCount = getExistingPageCount(view);
                             const triggerUpdate = (_footerHeight) => {
                                 requestAnimationFrame(() => {
@@ -258,8 +251,8 @@ export const PaginationPlus = Extension.create({
                                 triggerUpdate();
                                 return;
                             }
-                            const headerHeight = getHeaderHeight(view.dom, getCustomPages(this.options.customHeader, {}), "content");
-                            const footerHeight = getFooterHeight(view.dom, getCustomPages({}, this.options.customFooter), "content");
+                            const headerHeight = getHeaderHeight(view.dom, getCustomPages(opts.customHeader, {}), "content");
+                            const footerHeight = getFooterHeight(view.dom, getCustomPages({}, opts.customFooter), "content");
                             const footerHeightForCurrentPages = new Map();
                             for (let i = 0; i <= pageCount; i++) {
                                 if (footerHeight.has(i)) {
@@ -289,7 +282,7 @@ export const PaginationPlus = Extension.create({
                             for (const page of pagesSetToCheck) {
                                 const headerHeight = headerHeightForCurrentPages.has(page) ? headerHeightForCurrentPages.get(page) || 0 : headerHeightForCurrentPages.get(0) || 0;
                                 const footerHeight = footerHeightForCurrentPages.has(page) ? footerHeightForCurrentPages.get(page) || 0 : footerHeightForCurrentPages.get(0) || 0;
-                                const { _pageHeaderHeight, _pageHeight } = getHeight(this.options, headerHeight, footerHeight);
+                                const { _pageHeaderHeight, _pageHeight } = getHeight(opts, headerHeight, footerHeight);
                                 const contentHeight = page === 1 ? _pageHeight + _pageHeaderHeight : _pageHeight;
                                 if (page === 1) {
                                     pageContentHeightVariable[`rm-page-content-first`] = `${contentHeight}px`;
@@ -347,65 +340,74 @@ export const PaginationPlus = Extension.create({
     addCommands() {
         return {
             updatePageBreakBackground: (color) => () => {
-                this.options.pageBreakBackground = color;
+                this.storage.pageBreakBackground = color;
                 return true;
             },
             updatePageSize: (size) => () => {
-                this.options.pageHeight = size.pageHeight;
-                this.options.pageWidth = size.pageWidth;
-                this.options.marginTop = size.marginTop;
-                this.options.marginBottom = size.marginBottom;
-                this.options.marginLeft = size.marginLeft;
-                this.options.marginRight = size.marginRight;
+                this.storage.pageHeight = size.pageHeight;
+                this.storage.pageWidth = size.pageWidth;
+                this.storage.marginTop = size.marginTop;
+                this.storage.marginBottom = size.marginBottom;
+                this.storage.marginLeft = size.marginLeft;
+                this.storage.marginRight = size.marginRight;
                 return true;
             },
             updatePageWidth: (width) => () => {
-                this.options.pageWidth = width;
+                this.storage.pageWidth = width;
                 return true;
             },
             updatePageHeight: (height) => () => {
-                this.options.pageHeight = height;
+                this.storage.pageHeight = height;
                 return true;
             },
             updatePageGap: (gap) => () => {
-                this.options.pageGap = gap;
+                this.storage.pageGap = gap;
                 return true;
             },
             updateMargins: (margins) => () => {
-                this.options.marginTop = margins.top;
-                this.options.marginBottom = margins.bottom;
-                this.options.marginLeft = margins.left;
-                this.options.marginRight = margins.right;
+                this.storage.marginTop = margins.top;
+                this.storage.marginBottom = margins.bottom;
+                this.storage.marginLeft = margins.left;
+                this.storage.marginRight = margins.right;
                 return true;
             },
             updateContentMargins: (margins) => () => {
-                this.options.contentMarginTop = margins.top;
-                this.options.contentMarginBottom = margins.bottom;
+                this.storage.contentMarginTop = margins.top;
+                this.storage.contentMarginBottom = margins.bottom;
                 return true;
             },
             updateHeaderContent: (left, right, pageNumber) => () => {
                 if (pageNumber) {
-                    this.options.customHeader = Object.assign(Object.assign({}, this.options.customHeader), { [pageNumber]: { headerLeft: left, headerRight: right } });
+                    this.storage.customHeader = Object.assign(Object.assign({}, this.options.customHeader), { [pageNumber]: { headerLeft: left, headerRight: right } });
                 }
                 else {
-                    this.options.headerLeft = left;
-                    this.options.headerRight = right;
+                    this.storage.headerLeft = left;
+                    this.storage.headerRight = right;
                 }
                 return true;
             },
             updateFooterContent: (left, right, pageNumber) => () => {
                 if (pageNumber) {
-                    this.options.customFooter = Object.assign(Object.assign({}, this.options.customFooter), { [pageNumber]: { footerLeft: left, footerRight: right } });
+                    this.storage.customFooter = Object.assign(Object.assign({}, this.options.customFooter), { [pageNumber]: { footerLeft: left, footerRight: right } });
                 }
                 else {
-                    this.options.footerLeft = left;
-                    this.options.footerRight = right;
+                    this.storage.footerLeft = left;
+                    this.storage.footerRight = right;
                 }
                 return true;
             },
         };
     },
 });
+const safeGetView = (editor) => {
+    try {
+        // editor.view.dom is a getter that throws if not mounted
+        editor.view.dom;
+        return editor.view;
+    } catch (err) {
+        return null;
+    }
+};
 const getExistingPageCount = (view) => {
     const editorDom = view.dom;
     const paginationElement = editorDom.querySelector("[data-rm-pagination]");
